content/400-pulse/100-what-is-pulse.mdx

```diff
@@ -6,23 +6,48 @@ tocDepth: 3
 toc: true
 ---
 
-<TopBlock>
+## What is Prisma Pulse?
 
-[Prisma Pulse](https://www.prisma.io/data-platform/pulse) is managed database-event infrastructure that captures and distributes your database events to your application. It simplifies subscribing to type-safe data changes with an extended [Prisma Client](/orm/prisma-client) to power real-time functionality.
+[Prisma Pulse](https://www.prisma.io/data-platform/pulse) is managed infrastructure that captures databases events (i.e. create, update and delete operations) and distributes them to your applications.
 
-</TopBlock>
+It lets you subsribe to any changes happening in your database using [Prisma Client](/orm/prisma-client) to power **real-time use cases** in your applications, such as:
```

Comment:
Could we also mention event-driven workflows or architectures? Either here, or another section if you find it more suitable.